queries.sql

```diff
@@ -16,3 +16,18 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT COUNT(escape_attempts), FROM animals GROUP BY neutered;
 SELECT MIN(weight_kg), MAX(weight_kg), species FROM animals GROUP BY species;
 SELECT AVG(escape_attempts), species FROM animals WHERE  date_of_birth < '01-01-2000' AND date_of_birth > '01-01-1990' GROUP BY species;
+
+
+SELECT name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+
+SELECT animals.name, owners.full_name FROM animals JOIN owners ON animals.owner_id = owners.id;
+
+SELECT COUNT(*), species.name FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
+
+SELECT COUNT(*) FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Jennifer Orwell';
+
```

Comment:
In this query you need to "List all Digimon owned by Jennifer Orwell" , so you need to specify the animal species also, and you can select animal name instead of count. Kindly fix this issue 👍  😄